com.github.bharatkalluri.gifup.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id" : "com.github.bharatkalluri.gifup",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
```

Comment:
You shouldn't use the master version on flathub.

Reply:
Shifted to 3.8